src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -382,11 +382,15 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
         proposals: List[(BlockHash, Option[ProtocolId])] <- tezosNodeOperator.getProposals(blocks.map(_.data))
         blockHashesWithProposals = proposals.filter(_._2.isDefined).map(_._1)
         blocksWithProposals = blocks.filter(blockData => blockHashesWithProposals.contains(blockData.data.hash))
-        (ballots, ballotCounts) <- {
+        ballotCounts <- Future.traverse(blocksWithProposals) { block =>
+          db.run(TezosDb.getBallotOperationsForCycle(TezosTypes.discardGenesis(block.data.metadata).level.cycle))
+            .map(block -> _)
+        }
+        ballots <- {
```

Comment:
I'm surprised that this check is needed, why passing an empty list of "block+proposals" might return something different from a Future empty list?

Reply:
I believe I put there before few changes as this fixed things, now I tested it and it works without this check